File: tests/fixtures/python/calls/service.py
from . import db


class UserService:
    def get_user(self, user_id):
        return db.query("SELECT * FROM users WHERE id = ?", user_id)

    def list_users(self):
        return db.query("SELECT * FROM users")


def fetch_users():
    svc = UserService()
    return svc.list_users()

File: tests/fixtures/typescript/calls/service.ts
import { query } from './db';

export class UserService {
    getUser(userId: number): Record<string, unknown> {
        return query("SELECT * FROM users WHERE id = ?", userId);
    }

    listUsers(): Record<string, unknown>[] {
        return query("SELECT * FROM users");
    }
}

export function fetchUsers(): Record<string, unknown>[] {
    const svc = new UserService();
    return svc.listUsers();
}
